lib/cli/kit/base_command.rb

```diff
@@ -31,6 +30,12 @@ def self.call(args, command_name)
         end
       end
 
+      def stats_tags(args)
+        tags = ["task:#{self.class}"]
+        tags << "subcommand:#{args.first}" if args&.first && has_subcommands?
```

Comment:
Not for this PR, but a thought that came to mind: should we be propagating `nil` forward in this `args` variable? Maybe we should default it to an empty array higher up the chain?

Reply:
I agree with the thought, but yeah, not in this PR